ilastik/utility/gui/widgets.py

```diff
@@ -60,3 +61,13 @@ def cleanup():
             f()
 
     return cleanup
+
+
+@contextmanager
+def silent_widget(widget: QWidget) -> QWidget:
```

Comment:
Nitpick: change to `blocked_signals(obj: QObject) -> QObject` to be more explicit, and more general in type annotations.